test/compression.js

```diff
@@ -9,6 +9,12 @@ var zlib = require('zlib')
 
 var compression = require('..')
 
+/**
+ * @const
+ * whether current node version has brotli support
+ */
+var hasBrotliSupport = process.versions ? 'brotli' in process.versions : false
```

Comment:
It looks like the code coverage only shows one of these branches are ever run. Under what Node.js versions would the other branch run?

Reply:
Yeah there were errors in the tests before, with process module. But I guess it was because I `require`d it. Removing the `require` changed the coverage, so it must be it. I'm cleaning up a bit now.